RaveBusinessLogic/TauDEM.xml

```diff
@@ -9,58 +9,48 @@
                 <Children collapsed="true">
                   <Node label="DEM" xpath="Raster[@id='DEM']" type="raster" symbology="dem" transparency="40" id="dem" />
                   <Node label="Channel Area" xpath="Geopackage/Layers/Vector[@id='CHANNEL_AREA']" type="polygon" id="channel_area" symbology="ChannelArea" />
-                  <Node label="Drainage Input(s)">
-                    <Children>
-                        <Node label="Channel Area" xpath="Geopackage/Layers/Vector[@id='CHANNEL_AREA']" type="polygon" id="channel_area" symbology="ChannelArea" />
-                        <Node label="Channel Line Network" xpath="" type="line" id ="" symbology="" />
-                    </Children>
-                    </Node>
                 </Children>
             </Node>
             <Node label="Intermediates" xpath="Intermediates">
                 <Children collapsed="true">
-                    <Node label="Rasterized Channel" xpath="Raster[@id='RASTERIZED_CHANNEL']" type="raster" symbology="rasterized_channel" transparency="40" />
+                    <Node label="Rasterized Channel" xpath="Raster[@id='RASTERIZED_CHANNEL']" type="raster" symbology="rasterizedchannel" transparency="40" />
                 </Children>
             </Node>
-            <Node label="Outputs" xpath="Outputs">
+            <Node label="Outputs">
                 <Children collapsed="false">
                     <Node label="TauDEM Outputs">
-                        <Children>
+                        <Children collapsed="false">
+                            <Node label="Slope Analysis (D∞)" xpath="Outputs/Raster[@id='DINFFLOWDIR_SLP']" type="raster" symbology="dinfflowdir_slp" transparency="40" />
+                            <Node label="DEM Hillshade" xpath="Inputs/Raster[@id='HILLSHADE']" type="raster" symbology="hillshade" id="hillshade" />
```

Comment:
I think the DEM Hillshade should be up in inputs (I know its a GDAL output), but it just makes more sense to pair it with DEM.